update/src/Pos/Update/Poll/Failure.hs

```diff
@@ -22,87 +22,121 @@ import           Pos.Crypto (shortHashF)
     | PollUnknownStakes !EpochIndex
-    | PollWrongSoftwareVersion { pwsvStored :: !(Maybe NumSoftwareVersion)
-                              ,  pwsvApp    :: !ApplicationName
-                              ,  pwsvGiven  :: !NumSoftwareVersion
-                              ,  pwsvUpId   :: !UpId}
-    | PollProposalIsDecided { ppidUpId        :: !UpId
-                           ,  ppidStakeholder :: !StakeholderId}
-    | PollExtraRevote { perUpId        :: !UpId
-                     ,  perStakeholder :: !StakeholderId
-                     ,  perDecision    :: !Bool}
-    | PollWrongHeaderBlockVersion { pwhpvGiven   :: !BlockVersion
-                                  , pwhpvAdopted :: !BlockVersion}
-    | PollBadBlockVersion { pbpvUpId       :: !UpId
-                            ,  pbpvGiven   :: !BlockVersion
-                            ,  pbpvAdopted :: !BlockVersion}
-    | PollTooLargeProposal { ptlpUpId  :: !UpId
-                           , ptlpSize  :: !Byte
-                           , ptlpLimit :: !Byte
-                           }
-    | PollMoreThanOneProposalPerEpoch { ptopFrom :: !StakeholderId
-                                      , ptopUpId :: !UpId
-                                      }
-    | PollUnknownAttributesInProposal { puapUpId  :: !UpId
-                                      , puapAttrs :: !UpAttributes
-                                      }
-    | PollTipMismatch { ptmTipDB     :: !HeaderHash
-                      , ptmTipMemory :: !HeaderHash
-                      }
+    -- PollWrongSoftwareVersion
+    --    pwsvStored
+    --    pwsvApp
+    --    pwsvGiven
+    --    pwsvUpId
+    | PollWrongSoftwareVersion
+          !(Maybe NumSoftwareVersion)
+          !ApplicationName
+          !NumSoftwareVersion
+          !UpId
+    -- | PollProposalIsDecided
+    --      ppidUpId
+    --      ppidStakeholder
+    | PollProposalIsDecided !UpId !StakeholderId
+    -- | PollExtraRevote
+    --       perUpId
+    --       perStakeholder
+    --       perDecision
+    | PollExtraRevote !UpId !StakeholderId !Bool
+    -- | PollWrongHeaderBlockVersion
+    --       pwhpvGiven
+    --       pwhpvAdopted
+    | PollWrongHeaderBlockVersion !BlockVersion !BlockVersion
+    -- | PollBadBlockVersion
+    --       pbpvUpId
+    --       pbpvGiven
+    --       pbpvAdopted
+    | PollBadBlockVersion !UpId !BlockVersion !BlockVersion
+    -- | PollTooLargeProposal
+    --      ptlpUpId
+    --      ptlpSize
+    --      ptlpLimit
+    | PollTooLargeProposal !UpId !Byte !Byte
+    -- | PollMoreThanOneProposalPerEpoch
+    --      ptopFrom
+    --      ptopUpId
+    | PollMoreThanOneProposalPerEpoch !StakeholderId !UpId
+    -- | PollUnknownAttributesInProposal
+    --       puapUpId
+    --       puapAttrs
+    | PollUnknownAttributesInProposal !UpId !UpAttributes
+    -- | PollTipMismatch
+    --       ptmTipDB
+    --       ptmTipMemory
+    | PollTipMismatch !HeaderHash !HeaderHash
```

Comment:
👍 I'm liking this new style with the comment-ed field names above the constructs.